src/pykiso/test_coordinator/test_execution.py

```diff
@@ -474,12 +475,18 @@ def execute(
 
         log_file_path = get_logging_options().log_path
         # TestRunner selection: generate or not a junit report. Start the tests and publish the results
+        # pykiso --junit
         if report_type == "junit":
-            junit_report_name = time.strftime(f"%Y-%m-%d_%H-%M-%S-{report_name}.xml")
-            project_folder = Path.cwd()
-            reports_path = project_folder / "reports"
-            junit_report_path = reports_path / junit_report_name
-            reports_path.mkdir(exist_ok=True)
+            report_path = Path(junit_path)
```

Comment:
Somehow you broke the old behaviour.

If I run 
pykiso -c .\examples\dummy.yaml --junit  
the report is now created at the location where pykiso has been started. Before those files where in the
"reports" folder. I would expect that same behaviour if user dont specify any path...

Something like this 
pykiso -c .\examples\dummy.yaml --junit customname.xml  doesnt work at all.

pykiso --help also doesnt show the optional flag.

Reply:
ok, i fixed it